.github/workflows/ibis-backends.yml

```diff
@@ -144,6 +144,12 @@ jobs:
               - druid
             services:
               - druid
+          - name: oracle
```

Comment:
Since sqlalchemy 2.0 is required you probably want to stick this in that job instead of the primary backend job.